karmabot.rb

```diff
@@ -76,11 +76,8 @@ def handleChange(text,channel,user)
   return true
 end
 
-def handleFetch(text,channel,user)
+def doKarma(text,channel,user)
   puts "checking karma for '#{text}'"
-  if(!text.match(/^!karma\b/))
-    return false
-  end
   #chop off the '!karma'
```

Comment:
won't this line cause problems if it doesn't start with !karma? i know you already checked in handleFetch, but I think leaving in the check on line 81 is worth doing.

Reply:
It feels redundant with the check in `handleFetch` - `doKarma` won't be called _unless_ the message starts with `!karma`.

If you're worried about cutting off 7 characters if this is ever refactored and called from another source, I can wrap the `text = text[7..text.length]` in the same if statement.

Reply:
Reverted back to original check.